sysboundary/ionosphere.cpp

```diff
@@ -3860,7 +3867,7 @@ namespace SBC {
                   std::array<Real, 3> vNeighbours({0,0,0});
                   Real pressure = 0;
                   // Get moments from the nearest cells
-                  const vector<CellID> closestCells = getAllClosestNonsysboundaryCells(cellID);
+                  const vector<CellID>& closestCells = getAllClosestNonsysboundaryCells(cellID);
```

Comment:
Changes like this look scary but I guess you've verified them...?

Reply:
Seems to pass testpackage fine. I can't imagine const references causing any issue.

Reply:
Also, the only thing that is being done with this array next is that it is iterated over to index mpiGrid with its values. So indeed, completely ok.